{"category": "PytestAssertRewriteWarning", "filename": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", "lineno": 1277, "message": "Module already imported so cannot be rewritten: anyio", "$report_type": "WarningMessage", "when": "config", "location": ["/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", 1277, "_consider_importhook"]}
{"category": "PytestAssertRewriteWarning", "filename": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", "lineno": 1277, "message": "Module already imported so cannot be rewritten: anyio", "$report_type": "WarningMessage", "when": "config", "location": ["/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", 1277, "_consider_importhook"]}
{"pytest_version": "8.3.3", "$report_type": "SessionStart"}
{"nodeid": "tests57_75_5.py::test_matrix_of_single_number_equal_to_sum", "location": ["tests57_75_5.py", 17, "test_matrix_of_single_number_equal_to_sum"], "keywords": {"test_matrix_of_single_number_equal_to_sum": 1, "__wrapped__": 1, "tests57_75_5.py": 1, "Folder_75": 1, "": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.0034138932824134827, "start": 1738461501.6232572, "stop": 1738461501.6266716, "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "2c84cfb589c9426eb798ebc741b77841", "node": "<WorkerController gw0>"}
{"nodeid": "tests57_75_5.py::test_matrix_of_single_number_equal_to_sum", "location": ["tests57_75_5.py", 17, "test_matrix_of_single_number_equal_to_sum"], "keywords": {"test_matrix_of_single_number_equal_to_sum": 1, "__wrapped__": 1, "tests57_75_5.py": 1, "Folder_75": 1, "": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/matrixlib/defmatrix.py", "lineno": 194, "message": "IndexError: index 1 is out of bounds for axis 1 with size 1"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    @ignore_warnings", "    def test_matrix_of_single_number_equal_to_sum():", "        input_matrix = np.matrix([[16]])", ">       assert submatrix_with_particular_sum(input_matrix) == [[[16]]]"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q57/Phi_3_results_5/Folder_75/tests57_75_5.py", "lineno": 21, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    result.extend(find_submatrices(matrix, i, j, np.array([[matrix[i, j]]])))"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q57/Phi_3_results_5/Folder_75/generated_answer.py", "lineno": 15, "message": "in submatrix_with_particular_sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    result.extend(find_submatrices(mat, row + dr, col + dc, np.vstack((path, mat[row + dr, col + dc]))))"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q57/Phi_3_results_5/Folder_75/generated_answer.py", "lineno": 10, "message": "in find_submatrices"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    def __getitem__(self, index):", "        self._getitem = True", "    ", "        try:", ">           out = N.ndarray.__getitem__(self, index)", "E           IndexError: index 1 is out of bounds for axis 1 with size 1"], "reprfuncargs": {"args": [["self", "matrix([[16]])"], ["index", "(0, 1)"]]}, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/matrixlib/defmatrix.py", "lineno": 194, "message": "IndexError"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    @ignore_warnings", "    def test_matrix_of_single_number_equal_to_sum():", "        input_matrix = np.matrix([[16]])", ">       assert submatrix_with_particular_sum(input_matrix) == [[[16]]]"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q57/Phi_3_results_5/Folder_75/tests57_75_5.py", "lineno": 21, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    result.extend(find_submatrices(matrix, i, j, np.array([[matrix[i, j]]])))"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q57/Phi_3_results_5/Folder_75/generated_answer.py", "lineno": 15, "message": "in submatrix_with_particular_sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    result.extend(find_submatrices(mat, row + dr, col + dc, np.vstack((path, mat[row + dr, col + dc]))))"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q57/Phi_3_results_5/Folder_75/generated_answer.py", "lineno": 10, "message": "in find_submatrices"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    def __getitem__(self, index):", "        self._getitem = True", "    ", "        try:", ">           out = N.ndarray.__getitem__(self, index)", "E           IndexError: index 1 is out of bounds for axis 1 with size 1"], "reprfuncargs": {"args": [["self", "matrix([[16]])"], ["index", "(0, 1)"]]}, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/matrixlib/defmatrix.py", "lineno": 194, "message": "IndexError"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/matrixlib/defmatrix.py", "lineno": 194, "message": "IndexError: index 1 is out of bounds for axis 1 with size 1"}, null]]}, "when": "call", "user_properties": [], "sections": [], "duration": 0.0002146521583199501, "start": 1738461501.626873, "stop": 1738461501.627088, "extras": [], "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "2c84cfb589c9426eb798ebc741b77841", "node": "<WorkerController gw0>"}
{"nodeid": "tests57_75_5.py::test_matrix_of_single_number_equal_to_sum", "location": ["tests57_75_5.py", 17, "test_matrix_of_single_number_equal_to_sum"], "keywords": {"test_matrix_of_single_number_equal_to_sum": 1, "__wrapped__": 1, "tests57_75_5.py": 1, "Folder_75": 1, "": 1}, "outcome": "passed", "longrepr": null, "when": "teardown", "user_properties": [], "sections": [], "duration": 0.00024944450706243515, "start": 1738461501.6699176, "stop": 1738461501.6701682, "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "2c84cfb589c9426eb798ebc741b77841", "node": "<WorkerController gw0>"}
{"exitstatus": 2, "$report_type": "SessionFinish"}
